app/backend/src/couchers/models.py

```diff
@@ -653,3 +653,262 @@ class InitiatedUpload(Base):
     @hybrid_property
     def is_valid(self):
         return (self.created <= func.now()) & (self.expiry >= func.now())
+
+
+class Node(Base):
+    """
+    Node, i.e. geographical subdivision of the world
+    """
+
+    __tablename__ = "nodes"
+
+    id = Column(BigInteger, primary_key=True)
+
+    name = Column(String, nullable=False)
+    parent_node = Column(BigInteger, ForeignKey("nodes.id"))
```

Comment:
The naming of columns should be consistent with the rest of models. This should be `parent_node_id`, I believe.

Reply:
Sorry, this was the first one I did, and forgot to change.